conv.py

```diff
@@ -1,22 +1,45 @@
 import socket
-s=socket.socket(socket.AF_INET,socket.SOCK_STREAM)
-host='snoopy.mpi-inf.mpg.de'
-port=666
-pos=0
-s.connect((host,port))
+s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+HOST = 'snoopy.mpi-inf.mpg.de'
+PORT = 666
+s.connect((HOST, PORT))
 s.send(b"HELLO.\n")
 s.send(b"DOWNLOAD.\n:")
-data=b""
-with open ('conv.txt','wb') as f:
-    while 1:
-        msg=s.recv(1024)
-        if not msg:
-                   break
-        else:
-             data+=msg
-             #print(msg)
-#             f.write(msg)
-file=open("conv.txt",'rb+')
-file.seek(56)
-file.write(b"62531")
+data = b""
+while 1:
+       msg = s.recv(1024)
+       if not msg:
+                 break
+       else:
+            data += msg
+
+start = data.split(b'\n')
+helo = data.split(b'\n')[0]
+print(helo)
```

Comment:
print statements are for debugging..remove them once they have served the purpose.